Cannot install with dune-based build

On a fresh checkout, if I

```
make -j -f Makefile.jst compiler
make install
```

then I get

```
mkdir -p "/home/reisenberg/local/ocaml/ocaml/rae_out/bin"
mkdir -p "/home/reisenberg/local/ocaml/ocaml/rae_out/lib/ocaml"
mkdir -p "/home/reisenberg/local/ocaml/ocaml/rae_out/lib/ocaml/stublibs"
mkdir -p "/home/reisenberg/local/ocaml/ocaml/rae_out/lib/ocaml/compiler-libs"
/bin/install -c -m 644 \
  VERSION \
  "/home/reisenberg/local/ocaml/ocaml/rae_out/lib/ocaml"
make -C runtime install
make[1]: Entering directory `/usr/local/home/reisenberg/ocaml/ocaml/runtime'
/bin/install -c ocamlrun ocamlrund ocamlruni "/home/reisenberg/local/ocaml/ocaml/rae_out/bin"
/bin/install: cannot stat ‘ocamlrun’: No such file or directory
/bin/install: cannot stat ‘ocamlrund’: No such file or directory
/bin/install: cannot stat ‘ocamlruni’: No such file or directory
make[1]: *** [install] Error 1
make[1]: Leaving directory `/usr/local/home/reisenberg/ocaml/ocaml/runtime'
make: *** [install] Error 2
```

(My install prefix is `$(pwd)/rae_out`.)

Have I made a mistake? Or is there a step missing in the new instructions? Thanks!

--- a/HACKING.jst.adoc
+++ b/HACKING.jst.adoc
@@ -28,3 +28,7 @@ to build without running the testsuite, or:
     $ make -f Makefile.jst hacking
 
 to start a continuously polling build of the compiler.
+
+Note that the dune-based build system is entirely separate from the main
+Makefile, and so you will need to use `make -f Makefile.jst install` 
+to install after building.

